frontend/src/components/History/index.jsx

```diff
@@ -238,7 +259,7 @@ class History extends React.Component {
                 <CodebookLabelMenuContainer />
                 <ReactTable
                     data={history_data}
-                    columns={[...COLUMNS, ...metadataColumns.map((column, i) => {
+                    columns={[...this.COLUMNS(), ...metadataColumns.map((column, i) => {
```

Comment:
I don't think this is specific to this branch but leaving a comment here to document it. It looks like attempting to sort the history table based off a metadata column shifts theorizer of the left-hand columns but any metadata-related columns do not reorder.

Reply:
I'm not sure! This is really odd. I ran this in dev and it has the same issue.

Reply:
Closing this since we're addressing in a different branch